.github/workflows/release-artifacts.yml

```diff
@@ -91,7 +91,34 @@ jobs:
 
   build-sdist:
     name: "Build pypi distribution files"
-    uses: "matrix-org/backend-meta/.github/workflows/packaging.yml@v1"
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+
+      - name: Install tools
+        run: python -m pip install build twine
```

Comment:
I don't think you need `build` here, if we're using `poetry build` instead.

No objections to using `poetry build` instead of `python -m build` in the backend meta repo. (It should hopefully be equivalent anyway---I think build should invoke poetry-core via pyproject.toml?)

Reply:
For whatever reason `python -m build` wasn't installing the build deps iirc?